hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -1547,3 +1507,38 @@ def hpat_pandas_series_nunique_impl(self, dropna=True):
             return len(data_set) + 1
 
     return hpat_pandas_series_nunique_impl
+
+@overload_method(SeriesType, 'count')
+def hpat_pandas_series_count(self, level=None):
+    """
+    Pandas Series method :meth:`pandas.Series.count` implementation.
+    .. only:: developer
+       Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_count
+    Parameters
+    -----------
+    self: :obj:`pandas.Series`
+          input series
+    level:         *unsupported*
+    Returns
+    -------
+    :obj:`pandas.Series`
+         returns :obj:`pandas.Series` object
+    """
+
+    _func_name = 'Method count().'
+
+    if not isinstance(self, SeriesType):
+        raise TypingError(
+            '{} The object must be a pandas.series. Given: {}'.format(_func_name, self))
+
+    def hpat_pandas_series_count_impl(self, level=None):
+        # """
+        # Return number of non-NA/null observations in the object
+        # Returns number of unique elements in the object
+        # Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_count
+        # """
+        return numpy.count(self._data)
```

Comment:
I'm afraid numpy does not support this function. Please, check it
`module 'numpy' has no attribute 'count'`